app/assets/stylesheets/widgets/primary_column/career_blocks.scss

```diff
@@ -0,0 +1,389 @@
+.career-block-widget {
+  display: grid;
+  height: auto;
+  justify-content: start;
+  grid-template-rows: 300px;
+  @include qbal-slideUp-enter;
+
+  &--hidden {
+
+    // display: none;
+    // visibility: hidden;
+    // top: -99999px;
+    // position: absolute;
+    // left: -99999px;
+    .career-block-widget {
```

Comment:
@nicholasnadel same with this commented code

Reply:
good call